src/Microsoft.AspNetCore.Blazor.Server/ServerSideBlazorOptions.cs

```diff
@@ -1,12 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-namespace Microsoft.AspNetCore.Blazor
-{
-    /// <summary>
-    /// Options for Server-Side Blazor.
-    /// </summary>
-    public class ServerSideBlazorOptions
```

Comment:
I killed this because we're not doing anything with it yet, and there's a good chance that if we do add options in the future, they will mostly go to `AddServerSideBlazor` instead of `UseServerSideBlazor`. In any case it's unclear, so I'd prefer to drop it now and re-add something later when we know what it will be.